docker_test.go

```diff
@@ -1955,7 +1955,7 @@ func assertExtractedFiles(t *testing.T, ctx context.Context, container Container
 	}
 }
 
-func terminateContainerOnEnd(tb testing.TB, ctx context.Context, ctr Container) {
+func terminateContainerOnEnd(tb testing.TB, ctx context.Context, ctr testcontainers.Container) {
```

Comment:
We need to remove it from here, as it's duplicated. But then, there will be other places where this function is still used. We probably need to do this move last.

Reply:
Yeah, I'm moving everything temporarily to the testhelpers file, then we should be able to move this back here once the whole project is done. Still a few PRs to go 😅